mission_control/navigator_launch/nodes/network_listener.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+from sub8_alarm import single_alarm
+from twisted.internet import reactor
+import time
+
+
+class NetworkCheck(object):
```

Comment:
See naming comment above, I think It would be nice to make it clear that the KeepAlive system is one coherent system with two components just by looking at the names.